sormas-api/src/main/java/de/symeda/sormas/api/task/TaskIndexDto.java

```diff
@@ -58,6 +59,9 @@
 	public static final String TASK_STATUS = "taskStatus";
 	public static final String TASK_TYPE = "taskType";
 	public static final String CONTEXT_REFERENCE = "contextReference";
+	public static final String REGION = "region";
+	public static final String DISTRICT = "district";
+	public static final String CONTEXT_ENTITY_NAME = "contextEntityName";
```

Comment:
I don't quite get the purpose of this property (and the associated column in the task grid); the entity's name is already displayed in the "Associated link" column, so it's basically redundant and takes away space for other columns that provide new information. The new information that it provides is the event title, but that doesn't really match the column header ("Associated entity name") and it's really only useful for tasks that are linked to events. For that reason, I'd ask you to remove this column and property. If you need the event title information in the task grid, my suggestion would be to write an additional method that builds an event caption based on the event title and date (removing the disease and event investigation status) and use that in the "Associated link" column.

Reply:
Well, that's what i've been asked to do in #3279 
I do agree with you and I already proposed to add the event title to the associated link caption but the idea hasn't been retained.

Reply:
@MateStrysewske I confirm that this new column is the result of the specification process between @carolinverset and @bernardsilenou 

The review step shouldn't question the user story or, at least, the comment should be done in the issue and not the review (because @alexandre-gille is not the product owner and can only do what he was asked to do...)

Reply:
@ftavin The problem is that we currently don't have the capacities to review every issue that is created before external developers are picking it up; because those issues aren't part of our internal plannings, we don't really have another process in place than reviewing what's been done and checking whether it makes sense in the first place at the same time. This is definitely not a good process and we should revise it, but for that we also need the time to think about a better solution - you get the point. ;)

In this case, I've reviewed the pull request and realised that part of the specifications don't make a lot of sense to me which is why I issued the change request. I'll outsource this to the actual issue in the future because that definitely makes sense. In any case, I would still like to see this changed because I'm not really happy with the proposed solution in #3279 concerning the entity name column.

Reply:
It's removed, and the event title now is in the associated link caption.